bdk-jvm/lib/src/test/kotlin/org/bitcoindevkit/OfflinePersistenceTest.kt

```diff
@@ -40,4 +40,31 @@ class OfflinePersistenceTest {
             actual = addressInfo.address.toString(),
         )
     }
+
+    @Test
+    fun testPersistenceWithDescriptor() {
+        val connection = Connection(persistenceFilePath)
+
+        val descriptorPub = Descriptor(
+            "wpkh([9122d9e0/84'/1'/0']tpubDCYVtmaSaDzTxcgvoP5AHZNbZKZzrvoNH9KARep88vESc6MxRqAp4LmePc2eeGX6XUxBcdhAmkthWTDqygPz2wLAyHWisD299Lkdrj5egY6/0/*)#zpaanzgu",
```

Comment:
I grabbed this value from printing out the descriptor in [testPersistence](https://github.com/bitcoindevkit/bdk-ffi/blob/d74acfae722746a9add8314bb226fc94fd78cf62/bdk-jvm/lib/src/test/kotlin/org/bitcoindevkit/OfflinePersistenceTest.kt#L22C1-L42C6) since the value in [testDescriptorBip86](https://github.com/bitcoindevkit/bdk-ffi/blob/d74acfae722746a9add8314bb226fc94fd78cf62/bdk-jvm/lib/src/test/kotlin/org/bitcoindevkit/OfflineDescriptorTest.kt#L8-L18) didn't work (`84'` vs `86'`)